ivy/functional/frontends/numpy/mathematical_functions/floating_point_routines.py

```diff
@@ -35,6 +35,28 @@ def _nextafter(
     return ivy.nextafter(x1, x2, out=out)
 
 
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def _signbit(
+    x,
+    /,
+    out=None,
+    *,
+    where=True,
+    casting="same_kind",
+    order="K",
+    dtype=None,
+    subok=True,
+):
+    if dtype is None:
+        dtype = ivy.dtype(x)
+    zero = ivy.zeros_like(x, dtype=dtype)
+    return ivy.less(x, zero)
```

Comment:
My bad 😅, I forgot to mention that as will.
```suggestion
    return x < 0
```

I guess that would be a more efficient solution. I removed `dtype` as it's supposed to be handled by `ufunc`.

Also I'm not sure a bit about this implementation, as I've seen a case where the sign bit is toggled with the value 0 (`-0`). Can you research about how this might impact this implementation? I believe it might

Reply:
These are my findings when I tried `np.signbit` on my console

```py
>>> numpy.signbit(-0.0)
True
>>> numpy.signbit(0.0)
False
```

Reply:
And

```python
>>> numpy.signbit(-0)
False
```

Actually it was summarized here as well https://note.nkmk.me/en/python-numpy-sign-signbit-copysign/